sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServices.java

```diff
@@ -206,6 +206,10 @@ interface StreamAppendClient extends AutoCloseable {
     /** Append rows to a Storage API write stream at the given offset. */
     ApiFuture<AppendRowsResponse> appendRows(long offset, ProtoRows rows) throws Exception;
 
+    default long getInflightWaitSeconds() {
```

Comment:
nit: pretty self explanatory but, maybe adding a quick method doc would be good.